### ISSUE TITLE
building mpich with build_visit2_7_1 on redhat 5 fails

### ISSUE THREAD

**Issue by aowen87** [aowen87/BAR#616]
Rick Angelini reported that when he tried to build mpich with build_visit2_7_1 on a generic redhat 5 system it failed. Here is his e-mail:

Still no luck on just building Mpich.  I've never built mpich before, but I would *think* that it would be a no-brainer on a standard, vanilla Redhat5
system using gcc.     It's never, ever easy ..... 

$  ./build_visit2.7.1 --no-visit --no-thirdparty --console --mpich --stdout

[deleted output]

  CC       src/mpi/comm/intercomm_merge.lo
  CC       src/mpi/comm/comm_split_type.lo
  CC       src/mpi/comm/comm_group_failed.lo
  CC       src/mpi/comm/comm_reenable_anysource.lo
  CC       src/mpi/comm/comm_remote_group_failed.lo
  CC       src/mpi/datatype/address.lo
  CC       src/mpi/datatype/get_address.lo
  CC       src/mpi/datatype/get_count.lo
  CC       src/mpi/datatype/get_elements.lo
  CC       src/mpi/datatype/get_elements_x.lo
  CC       src/mpi/datatype/pack.lo
  CC       src/mpi/datatype/unpack.lo
  CC       src/mpi/datatype/pack_size.lo
  CC       src/mpi/datatype/status_set_elements.lo
  CC       src/mpi/datatype/status_set_elements_x.lo
src/mpi/datatype/status_set_elements_x.c: In function
'MPIR_Status_set_elements_x_impl':
src/mpi/datatype/status_set_elements_x.c:41: error: 'LLONG_MAX' undeclared (first use in this function)
src/mpi/datatype/status_set_elements_x.c:41: error: (Each undeclared identifier is reported only once
src/mpi/datatype/status_set_elements_x.c:41: error: for each function it appears in.)
make[2]: *** [src/mpi/datatype/status_set_elements_x.lo] Error 1
make[2]: Leaving directory `/home/angel/Software/VisIT/mpich-3.0.1'
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory `/home/angel/Software/VisIT/mpich-3.0.1'
make: *** [all] Error 2
MPICH build failed.  Giving up
Unable to build or install MPICH.  Bailing out.

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Eric Brugger
Original creation: 01/23/2014 11:54 am
Original update: 01/23/2014 04:48 pm
Ticket number: 1708
